http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20200506_label_pk.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="leonardo" author="azmina" id="label_pk">
+        <addPrimaryKey tableName="LABEL" columnNames="resourceId, resourceType, key"/>
```

Comment:
I'm fine with this change but I'm curious if there is a specific reason for using primary key over unique constraint? Kinda wish I did it in my last PR

Reply:
If we want to use `insertOrUpdate` on the label table (in order to change existing labels) there needs to be a primary key for the table. I slacked about it earlier this week, sorry if I wasn't clear!

Reply:
Yeah I think I realized that, makes sense 👍